### ISSUE TITLE
Please, keep railscasts open-source

### ISSUE THREAD

**Issue by benoitr** [ryanb/railscasts#51]
Hi Ryan,

We cannot see updated source code anymore since Railscasts offers pro subscription.
It could be an invaluable source to have it again, especially regarding the way you manage subscription (edit, cancel and authorization through it) which is not an easy topic for beginner.

Thank you so much for taking it into consideration.

**Comment by dfyx** [comment]
Absolutely!

**Comment by fayimora** [comment]
I agree with this. I was also trying to do the same for an app and was hoping to get tips here but wasn't able to :(

**Comment by vstabile** [comment]
+1, came here to check it out how you handle your restricted content.

**Comment by CharlyMTO** [comment]
Hi @ryanb , RailsCasts is now Free, please source code Free too thanks.
